src/test/java/com/stripe/BaseStripeTest.java

```diff
@@ -123,54 +123,48 @@
 				Mockito.<Map<String, Object>>any(),
 				Mockito.<Class<T>>any(),
 				Mockito.any(APIResource.RequestType.class),
-				Mockito.any(RequestOptions.class))).thenReturn(APIResource.GSON.fromJson(response, clazz));
+				Mockito.<RequestOptions>any())).thenReturn(APIResource.GSON.fromJson(response, clazz));
 	}
 
-	public static class ParamMapMatcher extends ArgumentMatcher<Map<String, Object>> {
+	public static class ParamMapMatcher implements ArgumentMatcher<Map<String, Object>> {
 		private Map<String, Object> other;
 
 		public ParamMapMatcher(Map<String, Object> other) {
 			this.other = other;
 		}
 
 		/* Treat null references as equal to empty maps */
-		public boolean matches(Object obj) {
-			if (obj == null) {
+		public boolean matches(Map<String,Object> paramMap) {
```

Comment:
The compared object is now cast to the generic type directly in the signature of the `matches` method.

Reply:
Nice improvement.